### ISSUE TITLE
CLI not reporting errors after updating to 15.6.3

### ISSUE THREAD

**Issue by romainmenke** [stylelint/stylelint#6899]
HI Guys.
Something wrong after update from 15.6.2 to 15.6.3 

```css
.test {
    margin-bottom: 20zrr;     <-- incorrect unit here
}
```
With **stylelint 15.6.2**
CLI and VSCode reported:
`Unexpected unknown unit "zrr"  unit-no-unknown`

With **stylelint 15.6.3**
CLI:
no errors reported
VSCode:
`Unexpected unknown unit "zrr" (unit-no-unknown)Stylelint[unit-no-unknown](https://stylelint.io/user-guide/rules/unit-no-unknown)`

_Originally posted by @ScorpAL in https://github.com/stylelint/stylelint/issues/6879#issuecomment-1576249573_

**Comment by romainmenke** [comment]
@ScorpAL I've moved this to an issue because it isn't easy to keep track if conversations in closed pull requests and the underlying problem might not be related to that pull request.

----------

Is it correct that your issue is that the CLI isn't reporting issues any more after upgrading to `15.6.3` ?

**Comment by azat-io** [comment]
The same problem:

```
> pnpm exec stylelint "**/*.{pcss,css}"

 ERR_PNPM_RECURSIVE_EXEC_FIRST_FAIL  Command was killed with SIGSEGV (Segmentation fault): stylelint **/*.{pcss,css}
```

**Comment by romainmenke** [comment]
Hi,

This should be resolved with the release of `15.7.0` : https://github.com/stylelint/stylelint/blob/main/CHANGELOG.md#1570

Can you let us know if this is resolved?

**Comment by ScorpAL** [comment]
Hi @romainmenke ,
Will let you know later today when my cache proxy will be in sync.

Thank you for update!

**Comment by ScorpAL** [comment]
Checked.

Resolved now. 

Thank you!
